go.mod

```diff
@@ -64,18 +64,18 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/pkg/profile v1.7.0
 	github.com/prometheus/client_golang v1.17.0
-	github.com/prometheus/procfs v0.12.0
+	github.com/prometheus/procfs v0.15.1
```

Comment:
I think this one is ok but @thaJeztah can you double-check it that might have incidence with https://github.com/moby/moby/blob/99edfd549ae7ceb986bf0275adead3f90156b8df/vendor.mod#L188